theme/themes/pastanaga/extras/main.less

```diff
@@ -645,6 +645,12 @@ body.has-toolbar-collapsed .mobile-menu {
   transition: transform 0.5s cubic-bezier(0.09, 0.11, 0.24, 0.91);
 }
 
+.summary-listing-item-header {
+  color: #007eb1 !important;
```

Comment:
please use less vars. Also, absolutely no `!important`. See https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity for more info on how to solve the problem.

Reply:
I have added a new variables in main.variables for the title color and fixed this, thank you